addons/account/models/account_tax.py

```diff
@@ -498,6 +498,22 @@ def _message_log(self, **kwargs):
 
             return super()._message_log(**kwargs)
 
+    def _get_message_create_valid_field_names(self):
+        valid_field_names = super()._get_message_create_valid_field_names()
+        if self.has_access(self._mail_post_access):
+            valid_field_names.add('pinned_at')
+        return valid_field_names
+
+    def _get_allowed_message_post_params(self):
+        params = super()._get_allowed_message_post_params()
+        if self.has_access(self._mail_post_access):
+            params.add('pinned_at')
+        return params
+
+    def message_post(self, *, body='', subject=None, message_type='notification', email_from=None, author_id=None, parent_id=False, subtype_xmlid=None, subtype_id=False, partner_ids=None, attachments=None, attachment_ids=None, body_is_html=False, pinned_at=None, **kwargs):
```

Comment:
Should be sufficient
```suggestion
    def message_post(self, *, body='', pinned_at=None, **kwargs):
```